hbase-common/src/test/java/org/apache/hadoop/hbase/util/TestBytes.java

```diff
@@ -386,14 +388,11 @@ public void testBinarySearch() {
 
   @Test
   public void testToStringBytesBinaryReversible() {
-    //  let's run test with 1000 randomly generated byte arrays
-    Random rand = new Random(EnvironmentEdgeManager.currentTime());
     byte[] randomBytes = new byte[1000];
     for (int i = 0; i < 1000; i++) {
-      rand.nextBytes(randomBytes);
+      Bytes.random(randomBytes);
```

Comment:
We do not need to use SecureRandom here? Anyway, not a big deal, can have another follow on issue to clean up the usage of Bytes.random. Maybe we could introduce a insecureRandom(byte[]) where we just use ThreadLocalRandom.

Reply:
> Maybe we could introduce a insecureRandom(byte[]) where we just use ThreadLocalRandom.

Sure, we can do that in this patch. Good idea. Let me update.